### ISSUE TITLE
Monitoring responses for a route not working properly with endTimestamp parameter

### ISSUE THREAD

**Issue by ruddi10** [bench-routes/bench-routes#361]
**Describe the bug**
Even if the route is having ping,jitter and  response time when we try to fetch for there graph then the response from the api call has their values as 0 creating inconsistency.

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to Monitoring 
2. Click on any route details for the one having ping,jitter and Response values.
3. You will see that No values alert.

**Expected behaviour**
We get the correct response from the api.

**Screenshots**
![Screenshot from 2021-03-15 14-23-20](https://user-images.githubusercontent.com/56596436/111131016-d849b180-859d-11eb-8f6e-012a00a78a0c.png)

![Screenshot from 2021-03-15 14-23-24](https://user-images.githubusercontent.com/56596436/111131042-dd0e6580-859d-11eb-9962-f8dc1868816d.png)

![Screenshot from 2021-03-15 14-51-53](https://user-images.githubusercontent.com/56596436/111131284-1cd54d00-859e-11eb-9bc6-af1843f89484.png)

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04

**Comment by ruddi10** [comment]
@Harkishen-Singh can you please have a look at this I think there is a problem with the api but not sure what do you think ??

**Comment by muskankhedia** [comment]
@ruddi10 The API calls are fine here, this is because no data is stored in the files for the respective route. This mainly occurs when we don't receive the desired data from the route or there is a faulty route. Also, this needs to be handled in a better way in the backend.

**Comment by ruddi10** [comment]
Yeah after opening this issue I saw that for some cases it working fine but for some its failing
